### DISCUSSION TITLE
如何让我自己的样式能被antd 自动生成的类似:where(.css-dev-only-do-not-override-17seli4) 包裹？

### DISCUSSION THREAD

**Discussion by zhangqhzz** [ant-design/ant-design#49135] [Q&A]
原因是我有在元素上添加ant-table-row-expand-icon ant-table-row-expand-icon-collapsed，antd4时能很好运行，现在升级到5,由于:where的存在，antd内部的ant-table-row-expand-icon在被我使用后不能起作用，应该如何进行？

**Comment by Wxh16144** [comment]
贴个代码吧，描述清楚你的预期和现状是什么样的，或者先提前看看有关 `:where` CSS 选择器文档

**Comment by zombieJ** [comment]
where 是降优先级的，你的样式盖不住估计是 style 文件或者样式先后顺序有关。你可以看看这个：
https://ant-design.antgroup.com/docs/react/compatible-style-cn#layer-%E9%99%8D%E6%9D%83

通过  `@layer` 把 antd 整体优先级降低，你的就能盖住了。
